src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
```

Comment:
[Use for loop for creating several objects of the same class](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)
For example, you need to create several users and write them in an array. In our case let’s say number of users is 3.
Bad example:
```
public class Main {
  private static final int NUMBER_OF_USERS = 3;
    
  public static void main(String[] args) {
      User[] users = new User[NUMBER_OF_USERS];
      User firstUser = new User();
      User secondUser = new User();
      User thirdUser = new User();
      users[0] = firstUser;
      users[1] = secondUser;
      users[2] = thirdUser;
  }
}
```
Improved example:
```
public class Main {
  private static final int NUMBER_OF_USERS = 3;
    
  public static void main(String[] args) {
      User[] users = new User[NUMBER_OF_USERS];
      for(int i = 0; i < users.length; i++) {
          users[i] = new User();
      }
  }
}
```